patterns/roks/variables.tf

```diff
@@ -281,6 +282,30 @@ variable "operating_system" {
   }
 }
 
+variable "use_private_endpoint" {
+  type        = bool
+  description = "Set this to true to force all cluster related api calls to use the IBM Cloud private endpoints."
+  default     = false
+}
+
+variable "verify_worker_network_readiness" {
+  type        = bool
+  description = "By setting this to true, a script will run kubectl commands to verify that all worker nodes can communicate successfully with the master. If the runtime does not have access to the kube cluster to run kubectl commands, this should be set to false."
+  default     = false
+}
+
+variable "ignore_worker_pool_size_changes" {
```

Comment:
We do not support autoscale at this pattern level, this can be hard-coded to false in config.tf